modules/objdetect/include/opencv2/objdetect/aruco_board.hpp

```diff
@@ -0,0 +1,301 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef __OPENCV_ARUCO_BOARD_HPP__
+#define __OPENCV_ARUCO_BOARD_HPP__
+
+#include <opencv2/calib3d.hpp>
```

Comment:
What dependencies are required on 5.x branch?
Only `opencv_3d` is available for now. Do we need new dependencies?

/cc @vpisarev

Reply:
reduce calib3 from aruco_board.hpp